Felienne/hedy
# [FEATURE] Enables admin to filter on username

**Description**
In this PR we introduce a new feature to the admin users view: username filtering. Enabling the admin to filter on a username substring in the users overview.

**Fixes**
This PR fixes #3195.

**How to test**
Make sure you are an admin and navigate to `/admin/users`. Verify that the new option is there and everything works as expected.
